cmd/kubeadm/app/cmd/alpha/certs.go

```diff
@@ -188,20 +192,24 @@ func addRenewFlags(cmd *cobra.Command, flags *renewFlags) {
 
 func renewCert(flags *renewFlags, kdir string, handler *renewal.CertificateRenewHandler) {
 	internalcfg, err := configutil.LoadOrDefaultInitConfiguration(flags.cfgPath, &kubeadmapiv1beta2.InitConfiguration{}, &flags.cfg)
-	kubeadmutil.CheckErr(err)
+	if err != nil {
+		return
```

Comment:
we might have to return the actual error from this function here.

Reply:
Fixed.